Assets/MRTK/Tests/PlayModeTests/BoundsControlTests.cs

```diff
@@ -2072,6 +2282,7 @@ public IEnumerator ManipulateViaAppBarFarInteraction()
 
             // deactivate the bounds control via app bar
             yield return hand.MoveTo(appBarButtonAfterScale);
+            yield return new WaitForSeconds(0.5f); // make sure animation does not cause interference
```

Comment:
Is there any way to avoid this WaitForSeconds, or to turn off animations, or something else?

WaitForSeconds calls in tests are really bad because they will slow down the overall test runs (i.e. these two changes together add a second that will always exist, even if they could run faster in the future). Worse, having them makes it super easy to copy paste going forward, which allows them to proliferate further.

Reply:
I agree WaitForSeconds can be problematic, and technically speaking the CI seems to pass even without that call. The reason I added that is when running the tests with editor open the animation seems to cause problems, which can be solved by the delay. I don't believe there is an easy way to disable the animation (i.e. smoothing) after looking at the code. The function calls are now removed and the CI still passed. I don't think having to wait shortly before pressing the button on a smoothed app bar is a bug, but it is not ideal in a test environment for sure.